galasa-parent/dev.galasa.framework.api.ras/src/main/java/dev/galasa/framework/api/ras/internal/common/QueryParameters.java

```diff
@@ -127,6 +127,18 @@ public Instant getSingleInstant(String queryParameterName, Instant defaultValue)
 		return returnedValue;
 	}
 
-	
+	public Instant getSingleInstantIfParameterNotPresent(String queryParameterName , String parameterToTest) throws InternalServletException {
```

Comment:
Separate this into two methods ? 
One to checkAtLeastOneQueryParameterPresent(param1,param2)
which throws an exception if one of those isn't present.

Then just use the getSingleInstant(...) method from the calling logic ?